bin/templates/cordova/lib/pluginHandlers.js

```diff
@@ -320,26 +320,33 @@ function generateAttributeError (attribute, element, id) {
 function getInstallDestination (obj) {
     var APP_MAIN_PREFIX = 'app/src/main';
 
-    if (obj.targetDir.startsWith('app/')) {
+    var targetDirArray = obj.targetDir.split('/');
+
+    if (targetDirArray[0] === 'app') {
```

Comment:
I think this way is the safest way to handle all forms:
- `app`
- `app/`
- `app/whatever`

Reply:
Agreed, would favor a test case that shows the actual problem you are solving here.

If a plugin target-dir would start with "app" but not one of the correct forms then it will not be a valid location anyway.